src/client/ws/Gateway.ts

```diff
@@ -20,6 +28,10 @@ export class Gateway {
 			log({ state: 'WS', message: 'Connected to API' });
 		});
 
+		this.socket.addEventListener("close", e => {
```

Comment:
```suggestion
		this.socket.addEventListener('close', e => {
```
Please install a eslint extension to your IDE of choice, or just run `yarn lint` before committing. I'm pretty sure we even have a git hook to run before committing. Try running `yarn prepare`